pkgs/desktops/gnome-3/misc/gnome-screensaver/default.nix

```diff
@@ -0,0 +1,98 @@
+{ stdenv
+, fetchgit
+, fetchurl
+, autoreconfHook
+, dbus-glib
+, glib
+, gnome-common
+, gnome-desktop
+, gnome3
+, gtk
+, gsettings-desktop-schemas
+, pkgconfig
+, intltool
+, pam
+, systemd
+, wrapGAppsHook
+}:
+
+let
+  pname = "gnome-screensaver";
```

Comment:
Can you move these inside the derivation attrset and get rid of the `name` attribute?

Reply:
This seems to be what most packages use so you can put `pname` in `updateScript`, is it really a problem?

Reply:
Ah I missed that, done.